core/templates/dev/head/galleries/gallery.html

```diff
@@ -21,7 +21,12 @@
   <script type="text/javascript">
     GLOBALS.LANGUAGE_CODES_AND_NAMES = JSON.parse(
       '{{LANGUAGE_CODES_AND_NAMES|js_string}}');
-    GLOBALS.CAROUSEL_SLIDES_CONFIG = JSON.parse('{{CAROUSEL_SLIDES_CONFIG|js_string}}');
+    GLOBALS.CAROUSEL_SLIDES_CONFIG = JSON.parse(
+      '{{CAROUSEL_SLIDES_CONFIG|js_string}}');
+    {% if explorations_list %}
```

Comment:
The issue that having it as a shell is that we have to do a lot of sending back and forth (plus manual decoding from the front end of the url). The hope was that since the backend already has the url, it can easily decode it and send the results in one go.